python/dask_cudf/dask_cudf/io/parquet.py

```diff
@@ -230,6 +230,7 @@ def read_partition(
         # Extract supported kwargs from `kwargs`
         read_kwargs = kwargs.get("read", {})
         read_kwargs.update(open_file_options or {})
+        read_kwargs["lazy"] = kwargs.get("lazy", None)
```

Comment:
Shouldn't `read_kwargs["lazy"]` always be set to `False`? Seems like using `lazy=True` within `dd/dask_cudf.read_parquet` would create a recursion error.

Reply:
Good point, fixed.